scripts/dockerDeleteExpiredAccessTokens.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# Copyright 2019 OCAD University
+# 
+# Licensed under the New BSD license. You may not use this file except in
+# compliance with this License.
+# 
+# You may obtain a copy of the License at
+# https://github.com/GPII/universal/blob/master/LICENSE.txt
+
+# This script provide as way to run the node script that deletes expired access
+# tokens (GPII App Installation Authorization records) from CouchDB using a
+# docker image of gpii-universal
+
+GPII_APP_DIR=${GPII_APP_DIR:-"/app"}
+GPII_COUCHDB_URL=${GPII_COUCHDB_URL:-"http://couchdb:5984/gpii"}
+DELETE_ALL=${DELETE_ALL:-""}
```

Comment:
As we chatted, let's explore the possibility of setting the default "DELETE_ALL" flag to `false`.

Reply:
Given @mrtyler's response on [PR 383](https://github.com/gpii-ops/gpii-infra/pull/383/files#r291167169), switching to an explicit `false` for a default is fine and can be used in the .yaml file in gpii-infra.

But, the actual optional argument for `deleteAccessTokens.js` is `--deleteAll`.  Like all optional arguments, it's either "`--deleteAll`" or missing (the empty string).  Hence, making the default argument "`false`" here entails translating it to the empty string, and I've added that logic.

Reply:
Actually, after thinking about it for a bit, it doesn't *entail* transforming to the empty string, since "`false`" will not be seen as "`--deleteAll`" within `deleteAccessTokens.js`, and it will run properly (it won't delete all of the access tokens).  Still it's somewhat cleaner to make the switch.